### DISCUSSION TITLE
N64 can't pair gamepad

### DISCUSSION THREAD

**Discussion by Sidecar940** [darthcloud/BlueRetro#137] [Q&A] [answered]
Hi!, thanks for your awesome work with this :)

But I have a problem, I'm working with an adapter for n64, I can't pair any gamepad when is conected to console, if I connect the ESP32 throught usb it pairs and works fine, I have checked the voltage on the VIM pin and I have 3.31V, do you have any idea why this happen?

I tried with 2 diferent esp32 and 2 consoles with the same result.

**Comment by darthcloud** [comment, marked as answer]
Take a picture of the wiring in which I can see the connection well.

**Reply by Sidecar940** [reply]
Here is a photo of my test board, the numbers on the green board are those of the db25, connected like this scheme:
https://github.com/darthcloud/BlueRetroHW/blob/master/DIY/BlueRetroDIY.pdf

In the photo I have connected the red to +5V pin, the black to GND and the green to IO19 of the ESP32

![photo_2022-01-11_22-47-54](https://user-images.githubusercontent.com/7070875/149026646-8a39f164-f66f-4537-8b9c-75e2c25fd8a3.jpg)

This test is with the n64 fw file.

I have tried connecting players 2, 3 and 4 as well

**Reply by darthcloud** [reply]
Ok the problem is the devkit regulator cant take 3.3v on the 5v input. You need n64 3.3v to go directly on the 3.3v pin.

Also dont trust color make sure its the right pin is you haven't already.

**Reply by darthcloud** [reply]
N64 schematic is wrong 3.3v should be on pin 6 of DB25

**Reply by Sidecar940** [reply]
Thanks, connecting the 3.3v on pin 6 works like a charm :)

In a future I will use a SMD Cable PCB, is the cords table ok or I need to modify something?

If I'm not wrong in the board the pin 6 is GND and the 7 is VIN

**Reply by darthcloud** [reply]
The passthrough cable PCB is VOUT on pin 6 so it should be ok.

On my custom BlueRetro DevKit I got a LDO with very low voltage drop which allow me to feed 3.3v on VIN. But the draw back is that in take 6V maximum witch forced me to use an LDO in the PSX/2 cables to drop 8V to 5V.

The LDO from official ESP devkit got a range of 5V-18V

**Reply by Sidecar940** [reply]
Thank you, I will try when my pcb arrives,

I have sent you a ko-fi for your help
